frontends/web/src/components/toggle/toggle.module.css

```diff
@@ -1,5 +1,6 @@
 .container {
   display: inline-block;
+  flex-shrink: 0;
```

Comment:
This fixes an edge case when the element before the toggle was too long, it would squeeze the toggle together.

The old fix was `.setting > label` (could remove this in 2 places)